dotnet/src/SemanticKernel.Abstractions/AI/TextCompletion/CompleteRequestSettings.cs

```diff
@@ -47,6 +47,13 @@ public class CompleteRequestSettings
     /// </summary>
     public IList<string> StopSequences { get; set; } = Array.Empty<string>();
 
+    /// <summary>
+    /// How many completions to generate for each prompt. Default is 1.
+    /// Note: Because this parameter generates many completions, it can quickly consume your token quota.
+    /// Use carefully and ensure that you have reasonable settings for max_tokens and stop. 
+    /// </summary>
+    public int ResultsPerPrompt { get; set; } = 1;
```

Comment:
Since this class lives in the abstractions namespaces I didn't want to introduce any validation here because the 1-128 range is a limitation for Azure and OpenAI only. Might worth think about validations abstractions later on.